### ISSUE TITLE
What defines skill in Gem TD?

### ISSUE THREAD

**Issue by nvs** [nvs/gem#122]
# What defines skill in Gem TD?

This is an honest discussion topic. The reason this is an important notion to consider is that if the skill ceiling is too low, then there is not much room for players to compete or display their prowess. Currently it is my belief that there is not enough room for a player's skill to shine in Gem. Before I touch upon that, let's first discuss how skill can be displayed in Gem.

Here is a non-exhaustive list of areas where skill can be shown:

- The ability to maze.
- Knowing the map.
- Planning ahead.
- Micromanaging towers.
- Making decisions quickly.
- The ability to adapt.
- Being able to read the game.

Well, let's look over these points quickly.

## Mazing

This is a core aspect of the map, right? This should be a defining feature that separates Gem from other non-mazing TD maps and allows players to demonstrate their skill. However, Gem ultimately fails in this regard. The layout is static. As a result all the mazes are known, as well as their pros and cons. About the only skill involved in Gem in regards to mazing is being able to find the time to adopt a known maze as your own. Now, you could ignore all that and just figure things out on your own. But essentially you'll be reinventing the wheel. There is no room left to truly innovate here, so essentially there is not much skill involved at all.

## Knowing Things

Yeah, knowing things helps. In fact, it helps a lot. Knowing the recipes. Knowing the structures. Knowing the creeps. Knowing the air rounds. All of it matters. And none of it is difficult to figure out. In fact, most people know just about everything that is important in Gem. Sure, some people go the extra mile. And yeah, sometimes the little things matter. But not as often as you'd think, trust me.

Still, there is not a lot of skill in simply knowing things. Time investment, sure. But skill? No, sorry.

## Planning

Planning ahead is a good skill to have. And yeah, it comes into play in Gem. But if you know your maze, and you understand the most effective strategies, then you don't really have to plan much at all. You pull on the luck machine repeatedly in essentially another attempt at having the 'perfect game'. It honestly does not take a lot of skill to reach this proficiency. As a result, Gem seems lackluster and perhaps boring to many due to its formulaic nature.

## Micromanagement

In most cases this refers to funneling kills to an imbalance structure (e.g. an Elder Slate), or a piece that comprises an imbalanced structure (e.g. a Poison Slate). It typically is done early game, when it is easier to ensure a kill goes to the proper target.

Now, in rare circumstances a player may use structures to ensure that all units are slowed (e.g. Sapphires and/or Emeralds), or to ensure that a poison is applied appropriately (e.g. Emeralds). This tends to be most useful in very early levels and on air levels.

Micromanagement will always have a role in the map (unless of course that ability is removed!). But in something like Gem it should never be a defining notion of a player's skill. Simple a means of rewarding players who want to go the extra mile.

## Being Quick

In Gem this honestly matters. Deciding what to keep. Placing your gems fast. Minimizing the time your builder wastes. When it comes down to it being quick is just a combined understanding of the above points. Sometimes being quick comes with risk, for example misplacing a gem, or having an early maze.

Now, being quick is good and all, but there is more to it, at least in my opinion. But at least we're getting somewhere. To sum it up, players who are skilled tend to be quicker. But being quick does not necessary mean they are skilled.

## Adaptability

Okay, this is probably the biggest display of skill in Gem that one can have. What do you do with bad luck? Almost anyone with sufficient skill can breeze through the game given good or great luck. But not everyone can adjust their maze properly, or recognize a weakness in their initial strategy or gem offerings and react accordingly. Truly I've found that the best of the best can recover from the worst possible situations in Gem. Sadly, in many races a player's adaptability generally amounts to naught.

## Reading the Game

Recognizing another player's general skill level in comparison to your own is important. Being able to determine if you are capable of catching or staying ahead of another player given varying amounts of luck is also relevant. When it comes down to it, yes, skilled players are more adept at reading the game proficiently. However, reading the game does not really help you directly; it merely helps you gauge your current situation against those you are facing.

## Conclusion

When you consider a game full of equally able players, there is currently little room to express one's skill. Which means that a race full of skilled players comes down primarily to one thing: luck. Now, skill does matter, of course. In fact, it matters a lot when bad luck is involved. When things get rough, your more skilled players rise to the occasion. The problem is, however, that the rewarding feeling one has when dealing with bad luck is generally overshadowed by the imbalances in the map.

What good is recovering from poor luck when another player gets an early Elder or Stone of Bryvx? The power spike can be insurmountable, especially when you're having a bad game, and it is no wonder that some skilled players will simply throw in the towel. They recognize the futility of their situation. And it can be frustrating, as singular moments of luck can essentially crown a victor before the race really unfolds. And that is the real problem: a gradual power increase due to skillful choices and decisions becomes moot when someone can get an imbalanced structure early.

Now, a skilled player doesn't have to care about the race at all. But let's face it: your more skillful players tend to be your more competitive players. Of course they probably care about the race. However, due to the imbalanced nature of Gem in its current form, it is highly unlikely that someone skilled will bother to stay unless they are ahead, having similar luck to the other players, or are simply a glutton for punishment.

In short, Gem needs a bit of work to make it so skill matters more again. Balancing the map is a good starting point. After that, who knows? I imagine increasing the replay value will play a large role in improving the situation. Only time will tell.

**Comment by nvs** [comment]
Primarily just my thoughts, when trying to wrap my head about the map a bit further. Not really needed, except to reference.
